### PR TITLE
chore: update snapshot version to 0.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## What this PR changes/adds

Update snapshot version to `0.1.0-SNAPSHOT`

## Why it does that

Our next release version will be `0.1.0`, thus the next snapshot will be `0.1.1-SNAPSHOT` and the automatic version bump mechanism currently only bumps the `patch` version.

## Further notes

- depends on https://github.com/eclipse-edc/GradlePlugins/pull/162

## Linked Issue(s)

Closes # <-- _insert Issue number if one exists_

_Please be sure to take a look at the [contributing guidelines](https://github.com/eclipse-edc/Connector/blob/main/CONTRIBUTING.md#submit-a-pull-request) and our [etiquette for pull requests](https://github.com/eclipse-edc/Connector/blob/main/pr_etiquette.md)._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@
 </p>
 
 The Eclipse Dataspace Connector provides a framework for sovereign, inter-organizational data exchange. It will
-implement the International Data Spaces Dataspace Protocol (DSP) as well as relevant protocols associated with GAIA-X. The
```
Comment (juliapampus) [inline README.md:51]: I think the `The` got lost :D

File: docs/developer/version-catalogs.md
Comment (juliapampus) [inline docs/developer/version-catalogs.md:1]: Do we plan to continuously update the docs files accordingly? Fine for me, just wondering.

**Review by juliapampus — APPROVED** [review]

**Review by ndr-brt — APPROVED** [review]

File: docs/developer/version-catalogs.md
Comment (paullatzelsperger) [reply docs/developer/version-catalogs.md:1]: that should be done by the auto-version-bump job

**Comment by paullatzelsperger** [comment]
depends on https://github.com/eclipse-edc/GradlePlugins/pull/162

**Pull request merged**